### ISSUE TITLE
New Achievements and Badges: Checked off X Tasks

### ISSUE THREAD

**Issue by beffymaroo** [HabitRPG/habitica#8663]
These new achievements and badges are ready to be implemented! These should be received when a player checks off the number of tasks indicated by each badge. The badge images and the names/mouseover text for the user achievement page can be found here:

https://trello.com/c/PMBVCiyv/4-checked-off-x-tasks-badges

**Comment by librarianmage** [comment]
I can work on this!

**Comment by TheHollidayInn** [comment]
Woot!

**Comment by librarianmage** [comment]
@beffymaroo, should the previous badge be replaced by the second one?

**Comment by librarianmage** [comment]
Is there a 'disabled' icon?

**Comment by lemoness** [comment]
@MathWhiz Sorry that you never got responses on this! They should all be separate badges, not replacing each other (like Beast Master, Mount Master, and Triad Bingo).

By "disabled icon" do you mean the achievement with the question mark that indicates that it hasn't been earned yet?

**Comment by librarianmage** [comment]
That was my question, but now all unearned achievements have the ? icon.

**Comment by lemoness** [comment]
Yes, all unearned achievements should have the gray badge with the question mark on it.

**Comment by beffymaroo** [comment]
@MathWhiz Are you still working on this issue? Please let us know when you have a chance. Thank you! :)

**Comment by librarianmage** [comment]
@beffymaroo I am not

**Comment by beffymaroo** [comment]
@MathWhiz Thank you for the update!

**Comment by fentie** [comment]
By "task" do you specifically mean ToDos, or do Dailies and Habits count too?

**Comment by TNychka** [comment]
Hey! I can take a look at this!

**Comment by saraolson** [comment]
Staff are looking at implementing new user achievements, so we're putting this issue on hold for now.

**Comment by paglias** [comment]
Is this still wanted given the new onboarding achievements @Tressley ?

**Comment by Tressley** [comment]
@paglias -- These are still on hold until we have a moment to look at achievements again. 👍
